core/src/main/java/com/rudderstack/android/sdk/core/RudderClient.java

```diff
@@ -653,7 +654,11 @@ public static void updateWithAdvertisingId(@Nullable String advertisingId) {
      *
      * @param deviceToken Push Token from FCM
      */
-    public void putDeviceToken(@Nullable String deviceToken) {
+    public static void putDeviceToken(@Nullable String deviceToken) {
```

Comment:
Can you change the function name from `putDeviceToken` to `setDeviceToken`?
Why we are using `Nullable` here?

Reply:
Changing the naming convention of `updateWithAdvertisingId` and `setAnonymousId` to `putAdvertisingId` & `putAnonymousId`